cadc-dali/build.gradle

```diff
@@ -28,6 +28,11 @@ dependencies {
     implementation 'org.opencadc:cadc-util:[1.6,)'
     implementation 'org.opencadc:cadc-uws:[1.0.4,)'
 
+    implementation 'org.apache.parquet:parquet-avro:1.15.0'
+    implementation 'org.apache.parquet:parquet-hadoop:1.15.0'
+    implementation 'org.apache.hadoop:hadoop-common:3.3.6'
+    implementation 'org.apache.hadoop:hadoop-mapreduce-client-core:3.3.6'
```

Comment:
are all these libs needed to read/write parquet? the last one seems out of scope...

in general, I prefer to declare dependencies a little more flexibly, eg

feeling lucky/lazy:
```
implementation 'org.apache.parquet:parquet-avro:[1.15.0,)'
```
slightly less risky:
```
implementation 'org.apache.parquet:parquet-avro:[1.15.0,2.0)'
```
even more risk averse:
```
implementation 'org.apache.parquet:parquet-avro:[1.15.0,1.16)'
```
Probably "slightly less risky" is a good general choice unless we know otherwise.